src/indent.c

```diff
@@ -958,7 +958,10 @@ get_breakindent_win(
 	if (regmatch.regprog != NULL)
 	{
 	    if (vim_regexec(&regmatch, line, 0))
-		bri += wp->w_briopt_list;
+		if (wp->w_briopt_list > 0)
+		    bri += wp->w_briopt_list;
+		else
+		    bri += (*regmatch.endp - *regmatch.startp);
```

Comment:
> bri += (*regmatch.endp - *regmatch.startp);

I think you want to use `bri = ...`, not adding to the currently calculated indent.

Reply:
what about showbreak and other stuff then? let me check it...

Reply:
yeah, likely this might be needed for some options, and for other not (like `shift:`) :(

Reply:
If you use `bri = ` then `set showbreak=>` interferes:
![image](https://user-images.githubusercontent.com/234774/126285099-62414dcf-06f9-4ef1-a357-d0204645e9c2.png)

Reply:
> yeah, likely this might be needed for some options, and for other not (like `shift:`) :(

With current implementation in this PR it works ok with or without showbreak:
![image](https://user-images.githubusercontent.com/234774/126285727-f42c1fbf-bbc2-4c8e-b36d-c6ffd52ca587.png)

Reply:
I take it back, it should not add to bri...

Reply:
Fixed with showbreak char in mind.